site/app/templates/submission/homework/LoadMessagePage.twig

```diff
@@ -0,0 +1,12 @@
+<div class="content gradeable_message">
+    <header id="gradeable-info">
+        <h1>
+            Message for: {{ gradeable_name }}
+        </h1>
+    </header>
+    <div>{{ load_gradeable_message | markdown }}</div>
+    </div>
+    <div class="content">
+    <a type="button" id="continue" href="{{ button_forward }}"class="btn btn-primary btn-lg" style="float: right">Accept, Continue to<br>'{{ gradeable_name }}'</a>
+    <a type="button" id="back" href="{{ button_back }}" class="btn btn-primary btn-lg">Cancel, Return to<br>Home Page</a>
```

Comment:
These tags are what @bmcutler is referencing.

Reply:
Ohh yeah this is most likely it `a` tags can't have a type, if you remove that and leave the class the same I think it'll fix it

Reply:
Yeah it looks like that fixed it - I tested on Safari on iPad along with Chrome on Windows and it changed the button to look normal on Safari after the change.